setup.cfg

```diff
@@ -0,0 +1,100 @@
+[metadata]
+name = sncosmo
+author = The SNCosmo Developers
+author_email = kylebarbary@gmail.com
+license = BSD 3-Clause License
+license_file = LICENSE.rst
+url = https://sncosmo.readthedocs.org
+description = Package for supernova cosmology based on astropy
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords = supernova, cosmology
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Topic :: Scientific/Engineering :: Astronomy
+    Topic :: Scientific/Engineering :: Physics
+
+[options]
+# We set packages to find: to automatically find all sub-packages
+packages =
+    sncosmo
+    sncosmo.tests
+zip_safe = False
+setup_requires =
+    setuptools_scm
+install_requires =
+    astropy>=1.3.0
+    extinction>=0.4.2
+    numpy>=1.13.3
+    scipy>=0.19.0
+python_requires = >=3.6.2
+
+[options.extras_require]
+oldestdeps = # Oldest versions supported, used for tests.
+    astropy==1.3.0
+    extinction==0.4.2
+    numpy==1.13.3
+    scipy==0.19.0
+test = # Required to run the test suite.
+    pytest
+    pytest-astropy
+    cython
+test_coverage = # Required for coverage tests.
+    coveralls
+    coverage
+    pytest-cov
+all = # Used by some parts of sncosmo, but not required.
+    iminuit
+    matplotlib
+    emcee
+    nestle
+docs = # Required to build the docs.
+    cython
+    numpy
+    matplotlib
+    scipy
+    extinction
+    astropy
+    sphinx
+    sphinx_gallery
+    sphinx_rtd_theme
+    pillow
+    iminuit
+    emcee
+    numpydoc
+
+[options.package_data]
+* = data/*, data/*/*, data/*/*/*, data/*/*/*/*, data/*/*/*/*/*, data/*/*/*/*/*/*
+sncosmo = sncosmo.cfg
+
+[coverage:run]
+source = sncosmo
+omit =
+   sncosmo/tests/*
+   sncosmo/conftest.py
+   sncosmo/__init__.py
+   sncosmo/version.py
```

Comment:
Are we still using `version.py`?

Reply:
After going through this review, I'm not convinced that `setuptools_scm` is making anything easier for us. I'm going to drop it and revert to the old method...

Reply:
I laugh that you talked yourself into dropping `setuptools_scm` while simultaneously convincing me that we should use it.